src/dynarec/rv64/dynarec_rv64_emit_shift.c

```diff
@@ -38,7 +38,7 @@ void emit_shl32(dynarec_rv64_t* dyn, int ninst, rex_t rex, int s1, int s2, int s
         SUBI(s5, s2, rex.w?64:32);
         NEG(s5, s5);
         SRL(s3, s1, s5);
-        ANDI(s5, s3, 1); // F_CF
+        ANDI(s5, s3, 1); // LSB == F_CF
```

Comment:
Why not moving that ̀`ANDI` inside the `IFX`?

Reply:
Doing that, we will need another ANDI below where the `s5` is used again.

Reply:
Ah no, it's used for OF also